Integrations/FeedTAXII/FeedTAXII_test.py

```diff
@@ -95,3 +96,13 @@ def test_fetch_indicators(self, mocker):
             with open('FeedTAXII_test/TestCommands/indicators_results.json', 'r') as exp_f:
                 expected = json.load(exp_f)
                 assert res == expected
+
+
+class TestTaxiiClient:
+    # verify that unassigned instance variables are still instantiated
+    def test__send_request(self):
+        from FeedTAXII import TAXIIClient
+        client = TAXIIClient()
+        with requests_mock.Mocker() as m:
+            m.post('http://www.example.com', text='response does not matter')
```

Comment:
lets also add a test which asserts the right auth params are initialized

Reply:
I think they should be initialized in all cases.

Reply:
... so I don't think this makes sense